File: Api/Api/Models/SeatModel.cs
namespace Api.Models
{
	public class SeatModel
	{
		public int Row { get; set; }
		public int Seat { get; set; }
		public double Price { get; set; }
		public bool IsVIP { get; set; }
		public bool isAvailable { get; set; }
		public string CustomerEmail { get; set; }
		public bool wasPaidFor { get; set; }
	}
}

File: cinema/src/app/models/seat.model.ts
export class Seat {
    row: number;
    seat: number;
    price: number;
    isVIP: boolean;
    isAvailable: boolean;
    customerEmail: string;
    wasPaidFor: boolean;
  
    constructor(row: number, 
      seat: number, 
      price: number, 
      isVIP: boolean, 
      isAvailable: boolean, 
      customerEmail: string, 
      wasPaidFor: boolean) {
      this.row = row;
      this.seat = seat;
      this.price = price;
      this.isVIP = isVIP;
      this.isAvailable = isAvailable;
      this.customerEmail = customerEmail;
      this.wasPaidFor = wasPaidFor;
    }
  }